config/core/resources/pingsource.yaml

```diff
@@ -322,6 +322,154 @@ spec:
                       description: 'SinkURI is the current active sink URI that has been
                           configured for the Source.'
                       type: string
+  - <<: *version
+    name: v1beta2
+    served: true
+    storage: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: 'PingSource describes an event source with a fixed payload produced on a specified cron schedule.'
+        properties:
+          spec:
+            type: object
+            description: 'PingSourceSpec defines the desired state of the PingSource (from the client).'
+            properties:
+              ceOverrides:
+                description: 'CloudEventOverrides defines overrides to control the
+                        output format and modifications of the event sent to the sink.'
+                type: object
+                properties:
+                  extensions:
+                    description: 'Extensions specify what attribute are added or
+                                overridden on the outbound event. Each `Extensions` key-value
+                                pair are set on the event as an attribute extension independently.'
+                    type: object
+                    additionalProperties:
+                      type: string
+                    x-kubernetes-preserve-unknown-fields: true
+              contentType:
+                description: 'ContentType is the media type of `data` or `dataBase64`. Default is empty'
+                type: string
+              data:
+                description: 'Data is data used as the body of the event posted to the sink. Default is empty.
+                        If set, the default value of `contentType` will be set to `text/plain`.
+                        Mutually exclusive with `dataBase64`.'
+                type: string
+              dataBase64:
+                description: 'DataBase64 is base64 encoded binary data used as the body of the event posted to the sink.
+                        Default is empty. If set, the default value of `contentType` will be set to `text/plain`.
```

Comment:
I believe we agreed to leave `contentType` empty

Reply:
fixed.